backend/crawler/crawler.py

```diff
@@ -6,40 +6,185 @@
 checks cancel_flag of it.
 If cancel_flag is True, it stops current job and set status to 'canceled'.
 """
+import os
+
+dir_path = os.path.dirname(__file__)
+driver_path = os.path.join(dir_path, '../../chromedriver')
 
 from django.core.exceptions import ObjectDoesNotExist
+from selenium import webdriver
+
+from ttrs.models import *
+
+sid = {'1학기': 1,
+       '2학기': 2,
+       '여름학기': 3,
+       '겨울학기': 4
+       }
+
+day = {'월': '2018-05-07T',
+       '화': '2018-05-08T',
+       '수': '2018-05-08T',
+       '목': '2018-05-08T',
+       '금': '2018-05-08T',
+       '토': '2018-05-08T',
+       '일': '2018-05-08T',
+       }
+
+semester = '1학기'
 
 
 def run(crawler):
-    # let # of total lectures is 1200
-    total_cnt = 1200
+    driver = webdriver.Chrome(driver_path)
+    driver.implicitly_wait(3)
+
+    driver.get('https://sugang.snu.ac.kr/sugang/cc/cc100.action')
+    driver.find_element_by_id('detail_button').click()
+    driver.find_element_by_xpath('//*[@id="srchOpenShtm"]/option[{}]'.format(sid[semester])).click()
+    driver.find_element_by_xpath('//*[@id="srchOpenSubmattCorsFg"]/option[2]').click()
+    driver.find_element_by_class_name('btn_search_ok').click()
+
+    total_cnt = int(driver.find_element_by_xpath('//*[@id="content"]/div/div[3]/div[1]/div[1]/h3/span').text)
+    total_page = (total_cnt // 10) + 1
```

Comment:
This is not valid calculation for `total page`.
When `total_cnt` is 100, `total_page` will be `(100//10)+1 = 11` is not equal to actual # of pages, which is 10.

You have to change this calculation to,
```python3
total_page = ((total_cnt-1) // 10 + 1
```